### ISSUE TITLE
Expanded hardware button support/mapping (0xtob/#58)

### ISSUE THREAD

**Issue by asiekierka** [asiekierka/nitrotracker#76]
>In order to provide the capability of a faster workflow, currently unused buttons and button combinations can be assigned to functions that will allow users to edit patterns and instruments without needing to use the stylus. This has been incredibly successful with LittleSoundDj and would be a significant improvement to NitroTracker.

>I would suggest having the screen swap effect the button focus. Thus, when a view is on the top screen it receives button focus, while being on the button screen leaves control to the stylus.

**Comment by deadpahn** [comment]
I'd caution using LSDJ as the blueprint though lol. Navigating it can feel like playing a maze game blind folded. Although I do agree, control navigation would be insane.

**Comment by M-O-Marmalade** [comment]
I think this sounds like an awesome idea! Although personally, I don't use MIDI controllers with NitrousTracker 🤷

**Comment by deadpahn** [comment]
> I think this sounds like an awesome idea! Although personally, I don't use MIDI controllers with NitrousTracker shrug

Yeah I have no clue how midi works in this. I tried a long time ago. There is a third party wifi server you need to run, and if on the OG/LITE you need to set your router to WEP encryption (lol) which is crackable in a matter of a day (probably faster these days). I think focusing on SYNC on one channel might be a better route for using external hardware.
